### PR TITLE
Fix registration test cases

### PR DESCRIPTION
**Pull request by shweta83**
### Problem Statement
Manual cherrypick for https://github.com/SatelliteQE/robottelo/pull/13974/files

### CODE REVIEW

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_registration.py

File: tests/foreman/ui/test_registration.py [hunk 1]
```diff
@@ -42,6 +49,7 @@ def test_positive_verify_default_values_for_global_registration(
     assert cmd['general']['location'] == 'Default Location'
     assert cmd['general']['capsule'] == 'Nothing to select.'
     assert cmd['general']['operating_system'] == default_os.title
+    assert cmd['general']['operating_system'] == ''
```
Comment (Griffin-Sullivan) [inline tests/foreman/ui/test_registration.py:52]: Why the double assertion?

**Review by Griffin-Sullivan — APPROVED** [review]
Looks like the one failure is due to `ansible-core requires a minimum of Python2 version 2.7 or Python3 version 3.5. Current version: 2.6.6`. I wonder if this is a RHEL 6 issue? My team doesn't do much with RHEL6 Content Hosts. Is this an error anyone else has seen?

File: tests/foreman/ui/test_registration.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/ui/test_registration.py]: There was a cherry-pick issue. Corrected it.

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_registration.py

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_registration.py

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_registration.py

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 5726
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/api/test_registration.py
Test Result : ============ 11 passed, 697 warnings, 1 error in 4265.99s (1:11:05) ============
```

**Review by shubhamsg199 — APPROVED** [review]

**Pull request merged**